sched/db_dump.cpp

```diff
@@ -209,32 +210,158 @@ int DUMP_SPEC::parse(FILE* in) {
     return ERR_XML_PARSE;
 }
 
+//
+// File streams
+//
+class OutputStream {
```

Comment:
Should the class names be uppercase?

Reply:
The coding standards are here: https://boinc.berkeley.edu/trac/wiki/CodingStyle  which I think stats that class names should be upper case.  @davidpanderson can confirm if needed.

Reply:
@TheAspens  Thank you! I'll check it out and tweak the classes.